npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -165,12 +165,12 @@ export async function getTempDir (): Promise<string> {
 }
 
 export async function getAngularCliModules (projectRoot: string) {
-  const angularCLiModules = [
-    '@angular-devkit/build-angular/src/utils/webpack-browser-config.js',
-    '@angular-devkit/build-angular/src/webpack/configs/common.js',
-    '@angular-devkit/build-angular/src/webpack/configs/styles.js',
-    '@angular-devkit/core/src/index.js',
-  ] as const
+  const angularCLiModules: [path: string, fallbackPath?: string][] = [
```

Comment:
Thanks for the PR! I'm at ng conf and was expecting to come home and open this PR after find out about this change. So I appreciate you already opening this up. 

I agree that having logic around sourcing the angular version would be a clearer way to do this instead of using a fallback. Usually I am not a fan of duplication but I think it would be fine in this instance since we don't really have a shared lib for things like this as is. We can leave a comment about this being the same code (or similar) to the `@cypress/schematic` and that we might want to eventually refactor this and move to a shared lib.

Reply:
Sounds good! I'll push an update with that.

Reply:
It turned out to be simpler and not so much a duplication because I forgot the schematic helper uses the virtual Tree, while here, we directly read from the file system.